packages/salesforcedx-utils-vscode/src/cli/commandOutput.ts

```diff
@@ -8,24 +8,25 @@
 import { CommandExecution } from './commandExecutor';
 
 export class CommandOutput {
-  private buffer = '';
+  private stdoutBuffer = '';
+  private stderrBuffer = '';
 
   public async getCmdResult(execution: CommandExecution): Promise<string> {
     execution.stdoutSubject.subscribe(realData => {
-      this.buffer += realData.toString();
+      this.stdoutBuffer += realData.toString();
     });
     execution.stderrSubject.subscribe(realData => {
-      this.buffer += realData.toString();
+      this.stderrBuffer += realData.toString();
```

Comment:
![updateoutputgoestostdderr](https://user-images.githubusercontent.com/7286437/39652083-38daa5c6-4fa1-11e8-9205-484fe88fe9ee.png)

When the CLI updates, the update messages go into stderr and the command output goes to stdout. So, if the exit code is okay, then we can just use stdout and ignore the CLI update messages in stderr.